Reply/app/src/main/res/values/type.xml

```diff
@@ -81,4 +81,11 @@
         <item name="android:textColor">@color/color_on_surface_emphasis_high</item>
     </style>
 
+    <style name="TextAppearance.Reply.EmailSubjectTitle" parent="TextAppearance.Material3.BodyLarge">
+        <item name="fontFamily">@font/work_sans</item>
```

Comment:
i'm thinking we should make this inherit from TextAppearance.Reply.BodyLarge instead of TextAppearance.Material3.BodyLarge, that way we can remove the duplicated fontFamily, lineHeight, and maybe even the textColor below, since it already uses color_on_surface_emphasis_high which should be pretty dark (in light mode).